src/System.Numerics.Vectors/src/System/Numerics/Vector.cs

```diff
@@ -4290,7 +4290,7 @@ internal static unsafe Vector<T> SquareRoot(Vector<T> value)
                     SByte* dataPtr = stackalloc SByte[Count];
                     for (int g = 0; g < Count; g++)
                     {
-                        dataPtr[g] = (SByte)Math.Sqrt((SByte)(object)value[g]);
+                        dataPtr[g] = unchecked((SByte)Math.Sqrt((SByte)(object)value[g]));
```

Comment:
Thanks for pointing that out, completely missed that.
I think there is value in the entire codebase being consistent. I agree that explicitly using unchecked is more valuable in some places than others but if you say that most of the codebase is checked (in debug builds anyway) you then have to define what a good reason for an exception is. Just my two cents. :)